### ISSUE TITLE
Jigasi Member no media

### ISSUE THREAD

**Issue by DoM1niC** [jitsi/jitsi-videobridge#1878]
*This Issue tracker is only for reporting bugs and tracking code related issues.*

Before posting, please make sure you check community.jitsi.org to see if the same or similar bugs have already been discussed. General questions, installation help, and feature requests can also be posted to community.jitsi.org.

## Description
Jigasi join but no Audio

## Current behavior
```
ll act as server
JVB 2022-04-27 23:51:19.426 INFORMATION: [74] [confId=bfbfe8799e9a38f5 epId=211 gid=-2 conf_name=relaxedstadiumsmanipulateeasily@conference.meet.3dns.eu] TlsServerImpl.notifyClientVersion#194: Negotiated DTLS version DTLS 1.0
JVB 2022-04-27 23:51:19.442 INFORMATION: [74] [confId=bfbfe8799e9a38f5 epId=211 gid=-2 conf_name=relaxedstadiumsmanipulateeasily@conference.meet.3dns.eu] TlsServerImpl.notifyAlertRaised#235: Alert raised: level=2, description=80, message=null cause=null org.jitsi.nlj.dtls.TlsServerImpl.notifyAlertRaised(TlsServerImpl.kt:226)
org.bouncycastle.tls.DTLSRecordLayer.raiseAlert(Unknown Source)
org.bouncycastle.tls.DTLSRecordLayer.fail(Unknown Source)
org.bouncycastle.tls.DTLSServerProtocol.abortServerHandshake(Unknown Source)
org.bouncycastle.tls.DTLSServerProtocol.accept(Unknown Source)
org.bouncycastle.tls.DTLSServerProtocol.accept(Unknown Source)
org.jitsi.nlj.dtls.DtlsServer.accept(DtlsServer.kt:45)
org.jitsi.nlj.dtls.DtlsServer.start(DtlsServer.kt:41)
org.jitsi.nlj.dtls.DtlsStack.start(DtlsStack.kt:150)
org.jitsi.videobridge.transport.dtls.DtlsTransport.startDtlsHandshake(DtlsTransport.kt:107)
java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
java.base/java.lang.Thread.run(Thread.java:829)

JVB 2022-04-27 23:51:19.447 SCHWERWIEGEND: [74] [confId=bfbfe8799e9a38f5 epId=211 gid=-2 conf_name=relaxedstadiumsmanipulateeasily@conference.meet.3dns.eu] DtlsServer.accept#52: Error during DTLS connection: org.bouncycastle.tls.TlsFatalAlert: internal_error(80)
JVB 2022-04-27 23:51:19.447 SCHWERWIEGEND: [74] [confId=bfbfe8799e9a38f5 epId=211 gid=-2 conf_name=relaxedstadiumsmanipulateeasily@conference.meet.3dns.eu] DtlsTransport.startDtlsHandshake#110: Error during DTLS negotiation, closing this transport manager
org.bouncycastle.tls.TlsFatalAlert: internal_error(80)
        at org.bouncycastle.tls.DTLSServerProtocol.accept(Unknown Source)
        at org.bouncycastle.tls.DTLSServerProtocol.accept(Unknown Source)
        at org.jitsi.nlj.dtls.DtlsServer.accept(DtlsServer.kt:45)
        at org.jitsi.nlj.dtls.DtlsServer.start(DtlsServer.kt:41)
        at org.jitsi.nlj.dtls.DtlsStack.start(DtlsStack.kt:150)
        at org.jitsi.videobridge.transport.dtls.DtlsTransport.startDtlsHandshake(DtlsTransport.kt:107)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalStateException: Invalid algorithm: null
        at org.bouncycastle.tls.crypto.impl.bc.BcTlsECDSA13Signer.generateRawSignature(Unknown Source)
        at org.bouncycastle.tls.DefaultTlsCredentialedSigner.generateRawSignature(DefaultTlsCredentialedSigner.java:53)
        at org.bouncycastle.tls.TlsUtils.generateServerKeyExchangeSignature(TlsUtils.java:2602)
        at org.bouncycastle.tls.TlsECDHEKeyExchange.generateServerKeyExchange(TlsECDHEKeyExchange.java:79)
        at org.bouncycastle.tls.DTLSServerProtocol.serverHandshake(Unknown Source)
        ... 9 more
JVB 2022-04-27 23:51:21.377 INFORMATION: [69] [confId=bfbfe8799e9a38f5 gid=-2 stats_id=Brandi-6zC conf_name=relaxedstadiumsmanipulateeasily@conference.meet.3dns.eu ufrag=csntk1g1mg3k70 epId=3b1091ce local_ufrag=csntk1g1mg3k70] Agent.setState#943: ICE state changed from Completed to Terminated.
JVB 2022-04-27 23:51:21.378 INFORMATION: [69] [confId=bfbfe8799e9a38f5 epId=3b1091ce local_ufrag=csntk1g1mg3k70 gid=-2 stats_id=Brandi-6zC conf_name=relaxedstadiumsmanipulateeasily@conference.meet.3dns.eu] IceTransport.iceStateChanged#341: ICE state changed old=Completed new=Terminated
JVB 2022-04-27 23:51:22.407 INFORMATION: [69] [confId=bfbfe8799e9a38f5 epId=211 local_ufrag=9d0et1g1mg3l3e gid=-2 conf_name=relaxedstadiumsmanipulateeasily@conference.meet.3dns.eu ufrag=9d0et1g1mg3l3e] Agent.setState#943: ICE state changed from Completed to Terminated.
JVB 2022-04-27 23:51:22.408 INFORMATION: [69] [confId=bfbfe8799e9a38f5 epId=211 local_ufrag=9d0et1g1mg3l3e gid=-2 conf_name=relaxedstadiumsmanipulateeasily@conference.meet.3dns.eu] IceTransport.iceStateChanged#341: ICE state changed old=Completed new=Terminated
```

## Possible Solution
Revert some code to a older Version of JVB

## Steps to reproduce
Updated to last Commits on JVB
# Environment details
- Jicofo (master)
- Jigasi (master)
- JVB (master)
- Web (master)
- OpenJDK 11.0.14.1 
- Asterisk 19.3.2
- NGINX Webserver (master)

**Comment by nils-ohlmeier** [comment]
Thanks for the report.

I just confirmed that is caused by https://github.com/jitsi/jitsi-videobridge/commit/621d73c08c2a55536da5e6e83effcd002b821c66

I'm looking into a workaround.

**Comment by DoM1niC** [comment]
Fixed by https://github.com/jitsi/jitsi-videobridge/pull/1879

**Comment by Neustradamus** [comment]
It has been fixed by #1884.
